package.json

```diff
@@ -171,6 +171,7 @@
     "rimraf": "^2.6.2",
     "rxjs": "^6.3.2",
     "source-map-support": "^0.5.3",
+    "underscore": "^1.9.1",
```

Comment:
Please use lodash instead of underscore. Mostly for the sake of consistency, since we are using it in all dependencies. But it also seems to be reacher in features and has significantly better performance.

Reply:
done